packages/owlbot-bootstrapper/cli/run-trigger-command.ts

```diff
@@ -47,6 +47,18 @@ export function getLanguageSpecificValues(language: string) {
   throw new Error('No language-specific container specified');
 }
 
+export function getMonoRepoNameAndOrg(repoToClone: string | undefined) {
```

Comment:
Please define the return type explicitly

Reply:
Also, this is probably better named as `parseRepoNameAndOrg` or `parseGitHubRepository` as the fact that it's a mono-repo is irrelevant.